apps/full-site-editing/tsconfig.json

```diff
@@ -27,12 +27,16 @@
 		"esModuleInterop": false,
 
 		// Some type helpers from our webpack/
-		"types": [ "fse" ],
+		"types": [ "fse", "jest" ],
 		"typeRoots": [ "./typings", "./node_modules", "../../node_modules" ],
 
 		// Preserve JSX, allows compatibility with wp-element jsx babel transforms
 		"jsx": "preserve"
 	},
 	"exclude": [ "node_modules" ],
+	// Had to add that in order for vscode to recognize the tests and apply the jest types
+	"include": [
+		"*.test.ts",
+	],
```

Comment:
Odd… omitting include should include everything. Are you sure this was necessary? Did you do `Restart TS Server`?

This would fail to match most other TypeScript we have in the project as well, which seems like a problem.

Reply:
Yeah, good point. It could indeed have been because of vscode glitch, I'll check it out again.

Reply:
If I remove that include, vscode's TS extension starts complaining:

![fooi](https://user-images.githubusercontent.com/81248/87079380-90db7c00-c1eb-11ea-9cdb-c90ad56cd0f5.png)

The interesting thing is that even with including only the test files, TS features are working fine for `block-helper.ts`.

Reply:
> Looks like an intermittent out-of-memory build error in the desktop job (filed as an issue in #43533):
> 
> ```
> $ check-npm-client && node ${NODE_ARGS:---max_old_space_size=8192} ./node_modules/webpack/bin/webpack.js --config client/webpack.config.js --display errors-only && yarn run build-languages-if-enabled
> events.js:292
>       throw er; // Unhandled 'error' event
>       ^
> Error: write EPIPE
> ```
> 
> Re-starting CI - hopefully that should get it to work.

Copy that :+1:, will keep that in mind for future PRs.